zio-http/src/main/scala/zio/http/DnsResolver.scala

```diff
@@ -254,8 +254,10 @@ object DnsResolver {
     )
   }
 
-  def configured(path: String = "zio.http.dns"): ZLayer[Any, zio.Config.Error, DnsResolver] =
-    ZLayer(ZIO.config(Config.config.nested(path))) >>> live
+  def configured(
```

Comment:
Maybe variable arguments instead of `NonEmptyChunk`?

```scala
def configured(path: String, paths: String*): ZLayer[Any, Error, DnsResolver] =
  ???
```